ani-cli

```diff
@@ -340,6 +344,8 @@ shift $((OPTIND - 1))
 # main #
 ########
 
+base_url=$(resolve_baseurl $base_url)
```

Comment:
You don't need to pass `$base_url` to `resolve_baseurl` because it doesn't use it.